data_access_api.py

```diff
@@ -104,17 +86,15 @@ def get_dataset_by_extent(self, product, product_type=None, platform=None, time=
         if longitude is not None and latitude is not None:
             query['longitude'] = longitude
             query['latitude'] = latitude
+        if crs is not None:
+            query['crs'] = crs

```

Comment:
This bit is crazy. I'll fix datacube code so you can pass those params straight through to `load`. Sorry.